doc/source/math/geocoordinates.rst

```diff
@@ -119,7 +119,7 @@ Example Usage
     PRINT spot:LNG.                 // Print 20
     
     PRINT spot:DISTANCE.            // Print distance from vessel to x
-    
+                                    // (same altitude is presumed)
```

Comment:
This is false.
It does not presume the spot is at the same altitude as the vessel.  It presumes the spot is at the terrain surface.